admin_tests/pre_reg/test_serializers.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from osf_tests.factories import DraftRegistrationFactory, ProjectFactory, AuthUserFactory, RegistrationFactory, RetractionFactory
+from osf.models import DraftRegistrationApproval
+from website.prereg.utils import get_prereg_schema
+from admin.pre_reg.serializers import get_approval_status
+from website.app import init_app
+
+pytestmark = pytest.mark.django_db
+
+@pytest.fixture(autouse=True, scope='session')
+def app_init():
+    init_app(routes=False, set_backends=False)
+
+@pytest.mark.django_db
+class TestGetApprovalStatus:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def project(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def draft_pending_approval(self, user):
+        return DraftRegistrationFactory(initiator=user, registration_schema=get_prereg_schema())
+
+    @pytest.fixture()
+    def draft_approved_and_registered(self, user, project):
+        draft = DraftRegistrationFactory(initiator=user, registration_schema=get_prereg_schema())
+        draft.approval = DraftRegistrationApproval(
+            meta={
+                'registration_choice': 'immediate'
+            }
+        )
+        draft.approval.state = 'approved'
```

Comment:
While these tests will pass because they all occur within a transaction, if someone reuses these fixtures and tries to filter or refresh from the database, these changes will not persist. Whenever modifying data on a model for a test it is best to `.save()` it.

Makes things more maintainable in the future if someone comes in and adds more tests.